addons/point_of_sale/models/pos_order.py

```diff
@@ -197,6 +197,20 @@ def _prepare_invoice_line(self, order_line):
             'product_uom_id': order_line.product_uom_id.id,
         }
 
+    def _prepare_invoice_lines(self):
```

Comment:
you'll have to rebase the branch to resolve the conflict with what has been done with customer notes

Reply:
yep